test/powershell/engine/Remoting/InvokeCommandRemoteDebug.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+﻿##
+## PowerShell Invoke-Command -RemoteDebug Tests
+##
+
```

Comment:
I don't know whether this work on IoT and Windows Core Server? If not, then we need to change the 'skipping'.